anesthetic/plot.py

```diff
@@ -246,6 +248,86 @@ def plot_1d(ax, data, *args, **kwargs):
     return ans
 
 
+def hist_1d(ax, data, *args, **kwargs):
+    """Plot a 1d histogram
+
+    This functions is a wrapper around matplotlib.axes.Axes.hist, with the
```

Comment:
This docstring isn't quite correct, as it's a wrapper around astropy's hist function

Reply:
Well, astropy's hist function is a wrapper around matplotlib's hist function. I wrote it this way, because I assume people are more familiar with the matplotlib hist function, so I documented astropy's hist function in more detail whereas for the rest I just point to matplotlib's hist function.

Reply:
Perhaps a statement saying "This functions as a wrapper around matplotlib.axes.Axes.hist, or astropy.visualization.hist if it is available"

Reply:
Guess that would be more complete.